packages/react/src/components/Geo/Geocoder/index.tsx

```diff
@@ -37,27 +37,26 @@ type GeocoderControl = IControl & {
  *   return <Geocoder />;
  * }
  */
-export const Geocoder = ({
-  position = 'top-right',
-  ...props
-}: GeocoderProps) => {
+export const Geocoder = (props: GeocoderProps) => {
   const { current: map } = useMap();
 
   /**
    * This logic determines whether the Geocoder exists as part of a Map component or if it is a standalone component.
-   * The `useControl` hook inside `ControlledGeocoder` from `react-map-gl` makes it easy to add a control to a map,
+   * The `useControl` hook inside `GeocoderControl` from `react-map-gl` makes it easy to add a control to a map,
```

Comment:
I changed the name since calling a React component "controlled" is distinct from a map control.

Reply:
Makes sense to me. Should we update the name of `StandaloneGeocoder` as well so we have a consistent naming pattern?

Reply:
I'm down with that in this PR. Any good ideas? Should it just be `GeocoderStandalone`?

Reply:
SGTM!